src/python/pants/backend/python/goals/pytest_runner_integration_test.py

```diff
@@ -231,7 +267,11 @@ def test() -> None:
             ),
         }
     )
-    extra_args = ["--pytest-version=pytest>=4.6.6,<4.7", "--pytest-lockfile=<none>"]
+    extra_args = [
+        "--pytest-version=pytest>=4.6.6,<4.7",
+        "--pytest-extra-requirements=[]",
```

Comment:
@benjyw this should fix the test failures CI was hitting (I had to install Python 2.7 locally for this test to not be skipped). This might be a little overkill since it's also removing the `pytest-cov` default dependency - based on the body of the test I thought it's likely OK, but let me know if not